source/changelog.txt

```diff
@@ -12,6 +12,14 @@ Release Notes
    :depth: 1
    :class: singlecol
 
+v2.2.10
+-------
+
+*Released June 24, 2024*
+
```

Comment:
(Detail)
I might recommend saying something like:
Contains internal enhancements and improvements. Just so that it's clear why we're not including any new features.

Reply:
WIll do.